### PR TITLE
feat(#210): Support passing message and message context to Container Ad…

### PR DESCRIPTION
**Pull request by manarhusrieh**
#210: Support passing message and message context to Container Adapter.

* Add new parameter context to ContainerAdapter.get method.
* Update the usages in bus instance and workflow registry to pass the message to get method.

### CODE REVIEW

File: packages/bus-core/src/workflow/registry/workflow-registry.ts [hunk 1]
```diff
@@ -182,7 +182,11 @@ export class WorkflowRegistry {
           try {
             let workflow: Workflow<WorkflowState>
             if (container) {
-              const workflowFromContainer = container.get(options.workflowCtor)
+              const workflowFromContainer = container.get(
+                options.workflowCtor,
+                message,
+                messageAttributes
```
Comment (adenhertog) [inline packages/bus-core/src/workflow/registry/workflow-registry.ts:188]: does this need to conform to the context interface?

```suggestion
                options.workflowCtor,
                {  message, messageAttributes }
```

File: packages/bus-core/src/workflow/registry/workflow-registry.ts [hunk 2]
```diff
@@ -345,7 +349,11 @@ export class WorkflowRegistry {
     })
     let workflow: Workflow<WorkflowState>
     if (container) {
-      const workflowFromContainer = container.get(workflowCtor)
+      const workflowFromContainer = container.get(
+        workflowCtor,
+        message,
+        attributes
```
Comment (adenhertog) [inline packages/bus-core/src/workflow/registry/workflow-registry.ts:355]: also here?

```suggestion
        workflowCtor,
        { message, messageAttributes: attributes }
```

**Comment by adenhertog** [comment]
You may also need to update `bus-configuration.ts:261` to accept the same contract. I'm not sure why it's currently redeclared inline, I'm a bad programmer 😆  but the signature should be changed to:

```
withContainer(container: ContainerAdapter): this {
```

File: packages/bus-core/src/workflow/registry/workflow-registry.ts (hunk 2 above)
Comment (manarhusrieh) [reply packages/bus-core/src/workflow/registry/workflow-registry.ts]: Resolved @adenhertog

File: packages/bus-core/src/workflow/registry/workflow-registry.ts (hunk 1 above)
Comment (manarhusrieh) [reply packages/bus-core/src/workflow/registry/workflow-registry.ts]: Resolved @adenhertog

**Comment by manarhusrieh** [comment]
@adenhertog I have fixed the comments along with using ContainerAdapter interface. Please recheck it again.

Thanks

**Pull request merged**